README.md

```diff
@@ -6,11 +6,11 @@ These are the required headers for each request:
 
 *   "**User-Agent**" - Please identify the system/product you are using to make this request.
     
-*   "**X-Auth-Date**" - The current unix epoch time as a string. 5 minute window.
+*   "**X-Auth-Date**" - The current unix epoch time as a string. 5 minute window.  If accessing from Python, use only the integer portion of the time value.  Note that JavaScript's time value from a Date's getTime() is compatible, but needs to be scaled down by 1,000 before also being converted to an integer.
```

Comment:
This is true, but I don't think this kind of language specific definition should be present in this generic documentation at the top-level. Maybe there's a way to phrase this that would be more generic? 

"The current unix epoch time (seconds since the Epoch) as a string"

That would hopefully signal to users they should use the rounded to the nearest second representation of epoch time which would solve the issue for both JS and Python users.